package/CHANGELOG

```diff
@@ -23,6 +23,9 @@ Enhancements
 
 * Add pickling support for Atom, Residue, Segment, ResidueGroup 
     and SegmentGroup. (PR #3953)
+* PDBReader now populates ts.data['tempfactor'] with the tempfactor for
```

Comment:
Newest entry first 🙃

Reply:
this is a daft rule that doesn't improve anything, but ok